btk/catalog.py

```diff
@@ -41,7 +41,7 @@ def name(self):
     def get_raw_catalog(self):
         return self._raw_catalog
 
-    def apply_selection_function(self, selection_function):
+    def apply_selection_function(self, selection_function, *kwargs):
```

Comment:
Is this for additional arguments or keyword arguments ?
Arguments unpacking is done with `*`, keyword args with `**` since it's a dictionary.

Reply:
That's right, I think I made the arguments optional afterwards.